doc/loggable.md

```diff
@@ -250,7 +250,7 @@ Now lets revert it to previous version:
 ``` php
 <?php
 // first check our log entries
-$repo = $em->getRepository('Gedmo\Loggable\Entity\LogEntry'); // we use default log entry class
+$repo = $em->getRepository(LogEntry::class); // we use default log entry class
```

Comment:
```suggestion
$repo = $em->getRepository(\Gedmo\Loggable\Entity\LogEntry::class); // we use default log entry class
```
Otherwise, we should import the class with `use`.